assets/css/omise-css.css

```diff
@@ -414,6 +414,21 @@ ul.omise-banks-list {
 	border: 4px solid #4e2e7f;
 }
 
+/** BAY **/
+.mobile-banking-logo.bay {
+	background: url('../images/bay.svg') #fec43b;
+	border-color: #fec43b;
+	border: 4px solid #fec43b;
```

Comment:
Could we add some left padding on the image? Seems they are too close to the selectors.

Reply:
this position is fixed in https://github.com/omise/omise-woocommerce/pull/249 
on my local branch still 4.16 🙏